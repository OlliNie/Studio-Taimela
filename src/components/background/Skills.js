import React from 'react';
import styles from './MobileBackground.css';
import { makeSkillList } from './makeSkillList';
import { languageSkills, certificateSkills } from '../../text/background';

export default function Skills() {
  return (
    <section className={styles.background}>
      <section className={styles.section}>
        <h3>Languages</h3>
        {makeSkillList(languageSkills)}
      </section>
      <section className={styles.section}>
        <h3>Certificates</h3>
        {makeSkillList(certificateSkills)}
      </section>
    </section>
  );
}
