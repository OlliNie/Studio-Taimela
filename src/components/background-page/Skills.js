import React from 'react';
import styles from './Skills.css';
import MakeSkillList from './MakeSkillList';
import { languageSkills, certificateSkills } from '../../text/background';

export default function Skills() {
  return (
    <section className={styles.skillbackground}>
      <section className={styles.section}>
        <h3>Languages</h3>
        <MakeSkillList skills={languageSkills} />
      </section>
      <section className={styles.section}>
        <h3>Certificates</h3>
        <MakeSkillList skills={certificateSkills} />
      </section>
    </section>
  );
}
