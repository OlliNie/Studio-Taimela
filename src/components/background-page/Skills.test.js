import React from 'react';
import { shallow } from 'enzyme';
import Skills from './Skills';

describe('Skills component', () => {
  it('renders all skills', () => {
    const wrapper = shallow(<Skills />);
    expect(wrapper).toMatchSnapshot();
  });
});
