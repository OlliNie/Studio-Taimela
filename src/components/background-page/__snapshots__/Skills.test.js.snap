// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills component renders all skills 1`] = `
<section
  className="skillbackground"
>
  <section
    className="section"
  >
    <h3>
      Languages
    </h3>
    <MakeSkillList
      skills={
        Array [
          Object {
            "bold": Array [
              0,
            ],
            "text": "English Native, born in the USA",
          },
          Object {
            "bold": Array [
              0,
            ],
            "text": "Finnish Mother tongue",
          },
          Object {
            "bold": Array [
              0,
            ],
            "text": "French Conversational (B2)",
          },
          Object {
            "bold": Array [
              0,
            ],
            "text": "German Conversational (B1)",
          },
        ]
      }
    />
  </section>
  <section
    className="section"
  >
    <h3>
      Certificates
    </h3>
    <MakeSkillList
      skills={
        Array [
          Object {
            "bold": Array [
              0,
            ],
            "text": "TOGAF Foundation",
          },
          Object {
            "bold": Array [
              0,
            ],
            "text": "ITIL Certified",
          },
          Object {
            "bold": Array [
              0,
              1,
            ],
            "text": "Prince 2 Foundation",
          },
          Object {
            "bold": Array [
              2,
              3,
            ],
            "text": "Trained in Balanced Scorecard",
          },
        ]
      }
    />
  </section>
</section>
`;
